shared/build.gradle.kts

```diff
@@ -197,6 +197,11 @@ val buildWebApp by tasks.creating(Copy::class) {
     val jsWebpack = "jsBrowserProductionWebpack"
 
     dependsOn(wasmWebpack, jsWebpack)
+    
+    // TODO could be removed after migration to Kotlin 2.0+
+    kotlin.wasmJs {
```

Comment:
feel free to migrate using 2.0.0 latest RC

Reply:
It's better to do separately, since may require updating some other things